model/countries/AU/AU-formatting-rules.yaml

```diff
@@ -0,0 +1,63 @@
+formatting-rules:
+  address:
+    - street-address-alternative-1
+    - separator: "\n"
+    - locality1
+    - separator: " "
+    - admin-area1
+    - separator: " "
+    - postal-code
+    - separator: "\n"
+    - country-name
+    - skip: country # redundunt with country-name
+    - skip: street-address # reduntunt with street-address-alternative-1
+
+  street-address-alternative-1:
+    - in-building-location
+    - separator: "  " # using 2 spaces
+    - building-location
+
+  in-building-location:
+    - unit
+    - separator: " "
+    - floor
+    - separator: " "
+
+  building-location:
+    - building
+    - separator: " "
+    - street
+
+
+examples:
+  - id: name
+    comment: |
+      This is an example of name.
+    attributes:
+      given-name: John
+      family-name: Smith
+    output:
+      name:
+        show: true
+        text: John Smith
+
+  - id: address
+    comment: |
+      This is an example of a full address in Australia.
+    attributes:
+      street: Main Street
+      building: 345
+      floor: Level 7
```

Comment:
Floor should just by 7 (because if a website asks for your floor, you would enter 7). The "Level" should go into the formatting instructions.